content/secrets/01-install-and-configure-tools/copilot.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Embedded tab content"
+disableToc: true
+hidden: true
+---
+
+### Install and setup prerequisites
+
+```bash
+# Install prerequisite packages
+sudo yum -y install nodejs python
+
+# Setting CDK Version
+export AWS_CDK_VERSION="1.91.0"
+
+# Install the Copilot cli
+sudo curl -Lo /usr/local/bin/copilot https://github.com/aws/copilot-cli/releases/latest/download/copilot-linux && sudo chmod +x /usr/local/bin/copilot
+
+# Setting environment variables required to communicate with AWS API's via the cli tools
+echo "export AWS_DEFAULT_REGION=$(curl -s 169.254.169.254/latest/dynamic/instance-identity/document | jq -r .region)" >> ~/.bashrc
```

Comment:
This step was missing in the create the workspace workshop. I added it there, so you should be able to remove it from this section now.

Reply:
Acknowledged.  Will remove.   This part will only install the copilot CLI (which they probably already have done earlier in the workshop, but i wanted to make the section standalone if they didnt want to run any of the other modules they could still go through this one besides doing the initial workspace setup.